recipes-kernel/linux/linux-intel-rt-acrn-uos_5.4.bb

```diff
@@ -0,0 +1,35 @@
+SUMMARY = "Linux Preempt RT Kernel with ACRN enabled (UOS)"
```

Comment:
Can we split it into .inc and .bb ? 
*.inc can also be included in 4.19 RT recipe as well. So we can avoid code duplication (in dunfell).

Reply:
for reduce code duplication purpose, it would need to refactor other kernel recipe too

Reply:
If you mean 4.19 RT recipe only,  then please do changes.

Reply:
there are other duplicates as well, see line for KERNEL_FEATURES and DEPENDS